src/presentation/dtos/device-dto.ts

```diff
@@ -1,25 +1,25 @@
-import {INewsDevice, TelemetricsDevice} from '../../model/entities/device'
-import {DeviceType} from '../../model/enums/device-type'
-import {StatusCode} from '../../model/enums/status-code'
+import { CoreDevice, Device, INewsDevice, TelemetricsDevice } from '../../model/entities/device'
+import { DeviceType } from '../../model/enums/device-type'
+import { StatusCode } from '../../model/enums/status-code'
 
-export interface DeviceDtoInterface {
-  type: DeviceType
-  id: string
-  name: string
-  isConnected: boolean
-  statusCode: StatusCode
-  statusMessage: string
-}
+// export interface DeviceDtoInterface {
+//   type: DeviceType
+//   id: string
+//   name: string
+//   isConnected: boolean
+//   statusCode: StatusCode
+//   statusMessage: string
+// }
```

Comment:
Uncommented code should be deleted.

The comment for not needing "DeviceDtoInterfaces" also applied for the interfaces `INewsdeviceDtoInterface` and `TelemetricsDeiveDtoInterface`.

Reply:
Do we have some linting tool to remind us of removing multiline comments and TODO comments?

Reply:
Currently, no. I haven't looked into whether one exists.